machines/Issuers/IssuersActions.ts

```diff
@@ -233,5 +233,15 @@ export const IssuersActions = (model: any) => {
     resetVerificationErrorMessage: model.assign({
       verificationErrorMessage: () => '',
     }),
+
+    sendDownloadingFailedToVcMeta: send(
+      (context: any) => ({
+        type: 'VC_DOWNLOADING_FAILED',
+        vcMetadata: context.vcMetadata,
```

Comment:
Same doubt - https://github.com/mosip/inji/pull/1393#discussion_r1578965354

Reply:
removed as we are not using anything from the context